docs/src/introduction/deploy-and-call-with-cli.md

```diff
@@ -0,0 +1,57 @@
+# Deploy and Call a Sway Contract With the CLI
+
+## Spin Up a Fuel node
+
+In order to deploy the contract, let's spin up a local Fuel node:
```

Comment:
Should say that this should be done in a separate tab, since it runs in the foreground.